data/json/LIXA/LIXA_monsters.json

```diff
@@ -0,0 +1,383 @@
+[
+  {
+    "id": "LIXA_eigenspectre_1",
+    "type": "MONSTER",
+    "name": { "str": "eigenspectre" },
+    "description": "This is the opposite of a human silhouette; every facet of every detail on display, with no shape to make sense of them.  You can see both sides of an ID badge on a white labcoat.  You can see every muscle and nerve in eyes blown wide with horror.",
+    "default_faction": "nether_player_hate",
+    "species": [ "NETHER_EMANATION" ],
+    "bodytype": "human",
+    "volume": "62500 ml",
+    "weight": "2 kg",
+    "hp": 120,
+    "speed": 130,
+    "material": [ "flesh" ],
+    "symbol": "E",
+    "color": "light_gray",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 8,
+    "melee_damage": [ { "damage_type": "pure", "amount": 0, "armor_penetration": 1 } ],
```

Comment:
I think pure damage ignores all armor already.  But I'd be interested to know if I'm wrong.

Reply:
I will let you know when I get to that part of the testing! I actually thought the damage type here had more to do with effects on your equipment, vis. cutting your clothes or frying your electronics, but I think that's in a different spot.